src/translations/pt/coreComponents.js

```diff
@@ -0,0 +1,611 @@
+        },
+        "cc-avatar-icon": {
+          title: "Ícone",
+          description:
+            "Os avatares podem ser usados com um ícone em vez de uma imagem para enfatizar áreas que não necessariamente têm (ou precisam) de uma imagem associada.",
+        },
+        "cc-avatar-bg": {
+          title: "Cores de fundo",
+          description:
+            "Quando usados com ícones ou texto, deve haver uma cor de fundo dos tokens de cor do sistema de design aplicada à forma do avatar. Certifique-se de que ícones e texto tenham uma relação de contraste suficiente com o fundo de acordo com o padrão WCAG AA.",
+        },
+      },
+    },
+    "cc-badge": {
+      title: "Crachá",
+      description:
+        "Crachás são elementos que representam o status de um objeto ou valor de entrada do usuário.",
+      checklist: {
+        "cc-badge-appearance": {
+          title: "Aparência",
+          description:
+            "Os crachás podem desempenhar vários papéis em seu produto e ter uma cor predefinida para cada papel deve ajudar os usuários a entender seu significado. Ao alterar as cores, certifique-se de que o texto tenha relação de contraste suficiente com o fundo de acordo com o padrão WCAG AA.",
+        },
+        "cc-badge-dismiss": {
+          title: "Ação de dispensar",
+          description:
+            "Os crachás podem ser usados como uma maneira dinâmica de exibir valores selecionados e deve haver uma maneira de dispensá-los.",
+        },
+      },
+    },
+    "cc-banner": {
+      title: "Banner",
+      description:
+        "Banners exibem uma mensagem acionável usada como forma proeminente de comunicação com seus usuários.",
+      checklist: {
+        "cc-banner-appearance": {
+          title: "Aparência",
+          description:
+            "Os banners são usados para exibir diferentes tipos de mensagens e é importante diferenciar sua aparência visual com base no papel que desempenham. Se estiver usando cores de fundo para diferenciação de papéis, certifique-se de que haja uma taxa de contraste suficiente com o conteúdo de acordo com o padrão WCAG AA.",
+        },
+        "cc-banner-image": {
+          title: "Área para ícones ou imagens",
+          description:
+            "Os banners podem complementar sua mensagem usando um ícone ou imagem de suporte. Eles não devem ser usados ​​em vez do conteúdo de texto.",
+        },
+        "cc-banner-actions": {
+          title: "Ações",
+          description:
+            "As ações nos banners devem estar relacionadas ao seu texto e fornecer uma maneira de reagir à mensagem enviada ao usuário.",
+        },
+        "cc-banner-dismiss": {
+          title: "Ação dispensável",
+          description:
+            "Não sobrecarregue o usuário com banners na página e inclua uma ação dispensável. Isso pode ser um botão separado de fechar ou uma das ações fornecidas.",
```

Comment:
In Portuguese, "dispensável" can bring a "not needed" feeling. I'd recommend something in the lines of:
"Ação de saída"
"...inclua uma ação de saída."